cmd/app/add_test.go

```diff
@@ -178,12 +178,12 @@ func TestAppAddCommand(t *testing.T) {
 							User     string "json:\"user,omitempty\""
 						}{},
 					},
-					types.SUCCESS,
```

Comment:
note: Since `types.SUCCESS` has no indication that it's a install success, I've prefixed all InstallState types to be `InstallSuccess`. I did the same for Permissions.

Reply:
100% agree. I'd like to see the `shared/types` package removed entirely. Instead, the types/interfaces/etc should belong to the package they're intended for. It would also allow for self-documentation, such as `install.Success`.